### PR TITLE
Day by day projection

### PR DESCRIPTION
**Pull request by yu-peng**
Separate the demand projection calculation from the add_estimate function. Now it can return a day-by-day projection in case we need it somewhere else.

Fixed the dates used in calculation: now assume that both start_date and end_date are inclusive.

### CODE REVIEW

**Comment by rcoh** [comment]
Awesome!! Excited to get this merged tomorrow and get new demand forecasts!

**Comment by mjacksonw** [comment]
> Make sure you git add the migration

@rcoh Even though the PR is approved, I assume this is a blocker for merging, so I'm holding off this am.

File: nyc_data/ppe/aggregations.py [hunk 1]
```diff
@@ -128,19 +123,59 @@ def add_demand_estimate(time_start: datetime,
...
+def get_projected_demand(time_start: datetime,
+                         time_end: datetime,
+                         demand_per_patient_per_day):
+
+    projected_demand = []
+
+    date = time_start
+    while date <= time_end:
+        hospitalization = HOSPITALIZATION[date.strftime("%Y-%m-%d")]
+        if not hospitalization or hospitalization < ALL_BEDS_AVAILABLE:
```
[the remark this answers is not in the document]
Comment (yu-peng) [reply nyc_data/ppe/aggregations.py:167]: Will move the hospitalization projection data to a spreadsheet and enable update through importer.

File: nyc_data/ppe/aggregations.py [hunk 2]
```diff
@@ -226,15 +250,29 @@ def render(self, value):
...
     asset = tables.Column()
-    projected_demand = NumericalColumn(accessor="demand", verbose_name="Demand Proxy", attrs={"th": {"class": "tooltip", "aria-label": "Demand projected based on the previous 7 days of hospital deliveries & IMHE hospitalization model"}})
-    balance = tables.Column(empty_values=(), order_by="percent_balance", attrs={"th": {"class": "tooltip", "aria-label": "Supply deficit or surplus against demand"}})
-
-    total = NumericalColumn(verbose_name="Supply", attrs={"th": {"class": "tooltip", "aria-label": "Sum of inventory, ordered, and made."}})
+    projected_demand = NumericalColumn(accessor="demand",
+                                       verbose_name="Demand Proxy",
+                                       attrs={
+                                           "th": {"class": "tooltip",
+                                                  "aria-label": "Demand projected based on the previous 7 days of hospital deliveries & IMHE hospitalization model"},
```
Comment (yu-peng) [inline nyc_data/ppe/aggregations.py]: Can we remove the IHME model part since we are using covidactnow model?

**Pull request merged**